src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -1475,7 +1445,7 @@ private int RefillByteBuff()
             else if (_currentEncoding is UnicodeEncoding ||
                 _currentEncoding is UTF32Encoding ||
                 _currentEncoding is ASCIIEncoding ||
```

Comment:
ASCIIEncoding is single byte, so can be removed from this as it's covered by `IsSingleByte()` already